<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debugger_main_window</class>
 <widget class="QMainWindow" name="debugger_main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1715</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>debugger_main_window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="minimumSize">
       <size>
        <width>1200</width>
        <height>800</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1715</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionHttp"/>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionAgent"/>
   </widget>
   <widget class="QMenu" name="menuNavigate">
    <property name="title">
     <string>Navigate</string>
    </property>
    <addaction name="actionGoto"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionRunThrough"/>
    <addaction name="actionRunNext"/>
    <addaction name="actionStop"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuNavigate"/>
   <addaction name="menuDebug"/>
   <addaction name="menuConfig"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAgent">
   <property name="text">
    <string>Agent</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>CloseAll</string>
   </property>
  </action>
  <action name="actionGoto">
   <property name="text">
    <string>Goto</string>
   </property>
   <property name="toolTip">
    <string>输入节点编号进行跳转</string>
   </property>
   <property name="whatsThis">
    <string>输入节点编号进行跳转</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
   <property name="toolTip">
    <string>根据内容进行查找</string>
   </property>
   <property name="whatsThis">
    <string>根据内容进行查找</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionRunThrough">
   <property name="text">
    <string>RunThrough</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionRunNext">
   <property name="text">
    <string>RunNext</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Shift+F5</string>
   </property>
  </action>
  <action name="actionHttp">
   <property name="text">
    <string>Http</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
